modules/ossm-federation-create-meshPeer.adoc

```diff
@@ -0,0 +1,86 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-federation-create-peer_{context}"]
+= Creating a ServiceMeshPeer resource
+
+.Prerequisites
+
+* Two or more {product-title} 4.6 or above clusters.
+* The clusters must already be networked.
+* The load balancers supporting the services associated with the federation gateways must be configured to support raw TLS traffic.
+* Each cluster must have a version 2.1 `ServiceMeshControlPlane` configured to support federation deployed.
+* An account with the `cluster-admin` role.
+
+////
+.Procedure from the Console
```

Comment:
Do we want to explicitly mention that creating the ServiceMeshPeer via the console currently isn't possible?

Reply:
At one point it was mentioned that the CRs might be part of the Operator?  I wrote this up with the idea that once the release went out I could test this and then publish it.   But if it's not going to happen in 2.1, it might be worth a mention?

Reply:
I thought you could create it in the console, but there wouldn't be any content.  @luksa, does there need to be an example resource in the csv file to make that happen?  I assumed the console would just create an empty resource and present the yaml editor.

Reply:
It's commented out at any rate, I just wrote it up in case it ended up going in so that Tim could verify and publish this later.